doc/examples/listings/iodhbwm-listings-color.tex

```diff
@@ -0,0 +1,92 @@
+% ---------------------------------------------------
+% Date:       12.12.2018
+% Version:    v0.1
+% Autor:      Felix Faltin <ffaltin91[at]gmail.com>
+% Repository: https://github.com/faltfe/iodhbwm
+% ---------------------------------------------------
+% --- --- --- --- -- Class options -- --- --- --- ---
+% ---------------------------------------------------
+\documentclass[
+    load-dhbw-templates, % Necessary for correct listing style
+    language = ngerman,
+    debug
+]{iodhbwm}
+\usepackage[T1]{fontenc}
+
+% Rename listings
+\renewcommand{\lstlistlistingname}{Quellcodeverzeichnis}

```

Comment:
These two lines are not relevant for this example, because they are covered by another feature.